src/modules.js

```diff
@@ -284,7 +284,7 @@ function cDefine(key) {
 }
 
 function isFSPrefixed(name) {
-  return name.length > 3 && name[0] === 'F' && name[1] === 'S' && name[2] === '_';
+  return name.length > 3 && name.startsWith('FS_');
```

Comment:
```suggestion
  return name.startsWith('FS_');
```

Reply:
(I don't think we need to ignore `FS_` itself here.)

Reply:
In that case I can just remove the function completely and inline it.